rbhl/api.py

```diff
@@ -0,0 +1,166 @@
+from collections import defaultdict
+from decimal import Decimal
+from opal.core.views import json_response
+from opal.core.api import LoginRequiredViewset, episode_from_pk
+from rbhl.models import Demographics
+from opal.core.api import OPALRouter
+
+
+class PeakFlowGraphData(LoginRequiredViewset):
+    base_name = "peak_flow_graph_data"
+
+    def day_to_dict(self, peak_flow_day, demographics, pef):
+        aggregates = peak_flow_day.get_min_max_variability_completeness()
```

Comment:
I might call this `.get_aggregate_data()`, return a dict and do `result.update(aggregates)` here...